client/get_transitions.go

```diff
@@ -0,0 +1,32 @@
+package client
+
+import (
+	"errors"
```

Comment:
Very minor, but it seems that your IDE has a long indentation.  Maybe you want to use space characters instead of tab characters for indentation. Common indentation for Java/Go is 4 spaces.  Nested code with long indentations can make the code less renderable and hard to read.  Again, this is a very minor issue, you don't need to fix it in this PR, but maybe something to consider for future changes.